PSKoans/Koans/IntroTraining/AboutCmdletVerbs.koans.ps1

```diff
@@ -0,0 +1,193 @@
+using module PSKoans
+[Koan(Position = 105)]
+param()
+<#
+    Cmdlet Verbs
+
+    The fundamental building block of PowerShell are cmdlets and functions.
+
+    We'll cover functions later, but cmdlets are made up of the following syntax:
+
+        Verb-Noun
+
+    With the verb being an action, and the noun representing the target of the command.
+
+    In this Koan, we'll cover the 5 basic verbs which'll see you through the majority of your
+    PowerShell needs. But PowerShell has more than just 5 predefined verbs! You can see them
+    all by running the following cmdlet:
+
+        Get-Verb
+#>
+
+Describe "Basic Verbs" {
+
+    Context "Get" {
+        <#
+            Cmdlets with the Get verb are used for retrieving data.
+
+            So for example "Get-Process" will retrieve all the processes running on 
+            your current machine.
+        #>
+
+        It 'is for commands that retrieve data'  {
+            <#
+                Using the Get-Command cmdlet, which retrieves all available commands,
+                find 5 new Get-* commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a Get cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb Get).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.Count -eq 5 | Should -BeTrue -Because "five unique cmdlets should be supplied"
+        }
+    }
+
+    Context "New" {
+         <#
+            Cmdlets with the New verb are used to create data.
+
+            So for example 'New-GUID' will create a new GUID, or 'New-LocalUser' will create a local
+            user on your machine.
+
+            Before continuing with this exercise, try using the following code to create a file:
+
+                $Path = "YOUR PATH.txt"
+                New-Item -Path $ItemPath -ItemType file
+
+            This will create a new text file, in the location you specify. 
+        #>
+
+        It 'is for commands that create data' {
+            <#
+                Using the Get-Command cmdlet, which all available commands,
+                find 5 New-* new commands you haven't previously used.
+            #>
+
+            # Replace each __ with the name of a New cmdlet
+            $Answers = "____","____","____","____","____"
+            $Answers | Should -BeIn (Get-Command -Verb New).Name
+
+            $UniqueCheck = $Answers | Get-Unique
+            $UniqueCheck.Count -eq 5 | Should -BeTrue -Because "five unique cmdlets should be supplied"
+        }
+
```

Comment:
Yup, that looks good! Just missed this one. 🙂 
```suggestion
```

Reply:
I'd hate to be in a meeting with you talking about variable naming conventions 👅